integration-tests/openshift-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.kubernetes-client.trust-certs=true
+quarkus.kubernetes-client.master-url=https://192.168.64.2:8443
+quarkus.kubernetes-client.namespace=mytest
+quarkus.kubernetes-client.username=developer
+quarkus.kubernetes-client.password=coucou
```

Comment:
I wonder if we want to keep the `kubernetes-client` prefix for the config but it's just a question.

Reply:
It would be nice if we could specify aliases for the `ConfigRoot` annotations

Reply:
The Config object is shared between `kubernetes-client` and `openshfit-client` so I am not really sure if it makes sense to use a different prefix. After all, they are all `kuberrnetes`.

Reply:
+1 for what Ioannis said